specification/cognitiveservices/data-plane/AzureOpenAI/inference/preview/2024-10-01-preview/inference.json

```diff
@@ -0,0 +1,12101 @@
+                  "enum": [
+                    "stop",
+                    "length",
+                    "content_filter"
+                  ]
+                },
+                "index": {
+                  "type": "integer"
+                },
+                "logprobs": {
+                  "type": "object",
+                  "nullable": true,
+                  "properties": {
+                    "text_offset": {
+                      "type": "array",
+                      "items": {
+                        "type": "integer"
+                      }
+                    },
+                    "token_logprobs": {
+                      "type": "array",
+                      "items": {
+                        "type": "number"
+                      }
+                    },
+                    "tokens": {
+                      "type": "array",
+                      "items": {
+                        "type": "string"
+                      }
+                    },
+                    "top_logprobs": {
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "additionalProperties": {
+                          "type": "number"
+                        }
+                      }
+                    }
+                  }
+                },
+                "text": {
+                  "type": "string"
+                },
+                "content_filter_results": {
+                  "$ref": "#/components/schemas/contentFilterChoiceResults"
+                }
+              }
+            }
+          },
+          "created": {
+            "type": "integer",
+            "description": "The Unix timestamp (in seconds) of when the completion was created."
+          },
+          "model": {
+            "type": "string",
+            "description": "The model used for completion."
+          },
+          "prompt_filter_results": {
+            "$ref": "#/components/schemas/promptFilterResults"
+          },
+          "system_fingerprint": {
+            "type": "string",
+            "description": "This fingerprint represents the backend configuration that the model runs with.\n\nCan be used in conjunction with the `seed` request parameter to understand when backend changes have been made that might impact determinism.\n"
+          },
+          "object": {
+            "type": "string",
+            "description": "The object type, which is always \"text_completion\"",
+            "enum": [
+              "text_completion"
+            ]
+          },
+          "usage": {
+            "$ref": "#/components/schemas/completionUsage"
+          }
+        },
+        "required": [
+          "id",
+          "object",
+          "created",
+          "model",
+          "choices"
+        ]
+      },
+      "createChatCompletionRequest": {
```

Comment:
Why were `parallel_tool_calls` and `stream_options` removed?

Reply:
Thanks for catching this! Fixed. 
The reason these two parameters were missing is that this PR (https://github.com/Azure/azure-rest-api-specs/pull/30849/files) was issued/merged after the 2024-10-01-preview base branch was created in private repo, and this change was not added to private repo change.